src/components/registration/steps.jsx

```diff
@@ -445,6 +436,13 @@ class DemographicsStep extends React.Component {
         this.setState({otherDisabled: gender !== 'other'});
     }
     handleValidSubmit (formData) {
+        // look up country name using user's country code selection
+        if (formData.countryCode) {
+            const countryInfo = countryData.lookupCountryInfo(formData.countryCode);
+            if (countryInfo) {
+                formData.user.country = countryInfo.name;
+            }
```

Comment:
If `countryInfo` is not found, then I think we haven't reached a valid state yet — that should be validated somewhere before this point.  If we do get to this point (where there is no `countryInfo`), then we should enter into an error state.

Reply:
Yes, this makes sense. Let me think about what sort of error state makes sense...

Reply:
OK, now it should pop up a validation error if it doesn't find a country name.